src/olympia/amo/context_processors.py

```diff
@@ -45,9 +45,6 @@ def global_settings(request):
 
         account_links.append({'text': ugettext('My Profile'),
                               'href': request.user.get_url_path()})
-        if request.user.is_artist:
-            account_links.append({'text': ugettext('My Themes'),
-                                  'href': request.user.get_themes_url_path()})
```

Comment:
> @eviljeff> "My Themes" link is going, right?  (I don't see it in new frontend.  Not sure "/user/eviljeffOo/themes" exists either)
@jvillalobos > yeah, that's going. There's an issue open for adding a My Add-ons link in that menu (Redundant with the devhub link, but it doesn't hurt).